rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
+channel = "1.56.1"
```

Comment:
Can you set these to whatever is up-to-date today before we merge?

Reply:
Done, bumped the version to today's stable.